graph/resolvers/asset.js

```diff
@@ -23,6 +23,7 @@ const Asset = {
 
     // Include the asset id in the search.
     query.asset_id = id;
+    query.statuses = ['NONE', 'ACCEPTED'];
```

Comment:
Should not be applied at the resolver level, instead should apply at the schema level. This prevents any future query to support a `statuses` field, which currently, the graph schema says it does allow.

Reply:
Mixed this up with the `RepliesQuery` as that one does not allow specifying statuses.